test/smoketest_artman.py

```diff
@@ -80,6 +80,8 @@ def run_smoke_test(root_dir, log):
     for msg in failure:
         logger.error(msg)
     if failure:
+        with open(log_file, 'r') as logs:
+            print(logs.read())
```

Comment:
Suggest a comment explaining why we need a print statement, since I don't think it's immediately obvious.